fcli-core/fcli-fod/src/main/java/com/fortify/cli/fod/issue/cli/cmd/FoDIssueUpdateCommand.java

```diff
@@ -0,0 +1,99 @@
+ * Copyright 2021, 2025 Open Text.
+ *
+ * The only warranties for products and services of Open Text 
+ * and its affiliates and licensors ("Open Text") are as may 
+ * be set forth in the express warranty statements accompanying 
+ * such products and services. Nothing herein should be construed 
+ * as constituting an additional warranty. Open Text shall not be 
+ * liable for technical or editorial errors or omissions contained 
+ * herein. The information contained herein is subject to change 
+ * without notice.
+ *******************************************************************************/
+
+package com.fortify.cli.fod.issue.cli.cmd;
+
+import java.util.ArrayList;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fortify.cli.common.output.cli.mixin.OutputHelperMixins;
+import com.fortify.cli.common.output.transform.IActionCommandResultSupplier;
+import com.fortify.cli.fod._common.cli.mixin.FoDDelimiterMixin;
+import com.fortify.cli.fod._common.output.cli.cmd.AbstractFoDJsonNodeOutputCommand;
+import com.fortify.cli.fod._common.util.FoDEnums.DeveloperStatusType;
+import com.fortify.cli.fod._common.util.FoDEnums.AuditorStatusType;
+import com.fortify.cli.fod._common.util.FoDEnums.VulnerabilitySeverityType;
+import com.fortify.cli.fod.issue.helper.FoDBulkIssueUpdateRequest;
+import com.fortify.cli.fod.issue.helper.FoDBulkIssueUpdateResponse;
+import com.fortify.cli.fod.issue.helper.FoDIssueHelper;
+import com.fortify.cli.fod.release.cli.mixin.FoDReleaseByQualifiedNameOrIdResolverMixin;
+import com.fortify.cli.fod.release.helper.FoDReleaseDescriptor;
+
+import kong.unirest.UnirestInstance;
+import lombok.Getter;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Mixin;
+import picocli.CommandLine.Option;
+
+@Command(name = OutputHelperMixins.Update.CMD_NAME)
+public class FoDIssueUpdateCommand extends AbstractFoDJsonNodeOutputCommand implements IActionCommandResultSupplier {
+    private static final Logger LOG = LoggerFactory.getLogger(FoDIssueUpdateCommand.class);
+    @Getter @Mixin private OutputHelperMixins.Update outputHelper;
+    @Mixin private FoDDelimiterMixin delimiterMixin; // Is automatically injected in resolver mixins
+    @Mixin private FoDReleaseByQualifiedNameOrIdResolverMixin.RequiredOption releaseResolver;
+
+    @Option(names = {"--user"}, required = true)
+    protected String user;
+    @Option(names = {"--dev-status"}, required = false)
+    protected DeveloperStatusType developerStatus;
+    @Option(names = {"--auditor-status"}, required = false)
+    protected AuditorStatusType auditorStatus;
+    @Option(names = {"--severity"}, required = false)
+    protected VulnerabilitySeverityType severity;
+    @Option(names = {"--comment"}, required = false)
+    protected String comment;
+    @Option(names = {"--ids"}, required = true, split=",")
+    protected ArrayList<String> vulnIds;
+
+    private long errorCount = 0;
```

Comment:
We should try to avoid having instance fields to avoid race conditions. In this case, `getActionCommandResult` should always return `ISSUES_UPDATED`, and `getJsonNode` should override this using `put(IActionCommandResultSupplier.actionFieldName, "ISSUES_UPDATED_WITH_ERRORS")`